src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -37,34 +42,31 @@ function Start-NewOrchestration {
         [Parameter(
             Position=2,
```

Comment:
IMPO, I believe that using a parameter by name makes more sense that having a positional parameter. This way is easier to understand what's the intention of the user.

Reply:
I think it's a good practice to _enable_ positional parameters for one or two most obvious things. For example, you wouldn't expect `Invoke-RestMethod` to require specifying the Uri parameter name - what else could it possibly be? And, if the name were mandatory, some users would struggle to remember: was it "Uri", "Url", or something else? I feel that `FunctionName` and `InputObject` here are in the same category: the user would intuitively expect these parameters in this order, without having to remember whether this is "FunctionName", or just "Function", or just "Name". This is also important because we may not have intellisense or autocomplete for these commands. And then it's up to the user whether to specify parameter names or not.

One thing I want to fix though is change Position=2 to 1. No idea why it was 2 - leftovers of legacy code.

Reply:
Ok, sounds good. One last thing: For consistency, should we ```OrchestrationClient``` position 3?